packages/reader-image-mode/src/ImageViewer.vue

```diff
@@ -88,6 +143,83 @@
           this.viewer.open([`${this.imageIdentifier}info.json`]);
         }
       },
+      async clearHighlights() {
+        await this.$store.dispatch(`${MODULE_NS}/${HIGHLIGHT_TRANSCRIPTION}`, {
```

Comment:
I changed this from a mutation to an action for consistency with other usage of Vuex in the project.